common/types/history.go

```diff
@@ -407,11 +407,11 @@ func (v *HistoryQueryWorkflowRequest) GetRequest() (o *QueryWorkflowRequest) {
 
 // HistoryQueryWorkflowResponse is an internal type (TBD...)
 type HistoryQueryWorkflowResponse struct {
```

Comment:
This is the type returned from History to Frontend, correct? Do we need to change it?